scripts/static-vdis

```diff
@@ -237,19 +242,18 @@ def attach(vdi_uuid):
                 except:
                     pass
                 path = None
-                if not (os.path.exists(d + "/sr-uri")):
+                if not (os.path.exists(d + "/" + smapiv3_config)):
                     # SMAPIv1
                     config = read_whole_file(d + "/config")
                     driver = read_whole_file(d + "/driver")
                     path = call_backend_attach(driver, config)
                 else:
                     volume_plugin = read_whole_file(d + "/volume-plugin")
-                    sr_uuid = read_whole_file(d + "/sr-uuid")
-                    sr_uri = read_whole_file(d + "/sr-uri")
+                    configuration = json.loads(read_whole_file(d + "/" + smapiv3_config))
```

Comment:
This is independent but, unless we need to support python 2.5, we could redefine `read/write_whole_file` Into more efficient and readable counterparts using the `io` module (https://docs.python.org/2/library/io.html), e.g.

```
Import io

def read_whole_file(filename):
    with io.open(filename,  mode="r", encoding="utf-8") as f:
        return f.read()
```